ui.apps/src/main/content/jcr_root/apps/core/cif/components/structure/navigation/v1/navigation/navigation.html

```diff
@@ -22,5 +22,5 @@
          data-sly-use.groupTemplate="group.html"
          data-sly-call="${groupTemplate.group @ navigationModel=navigationModel}">
     </nav>
-    <sly data-sly-resource="${@path='miniaccount'}"></sly>
-</aside>
\ No newline at end of file
+    <div id="miniaccount"></div>
```

Comment:
A div with this id is already added by the `miniaccount` component - https://github.com/adobe/aem-core-cif-components/blob/9ad38db585e48c320eb031c4dcfbfbc44737d9b6/ui.apps/src/main/content/jcr_root/apps/core/cif/components/content/miniaccount/v1/miniaccount/miniaccount.html#L14

You should keep the resource inclusion approach but check for the resource first.